lib/rebirth/base.rb

```diff
@@ -49,6 +54,14 @@ def to_hash
         hash[key] = __get_value(key).map{|v| klass.new(v).to_hash}
       end
 
+      self.class.table_attribute_methods.each do |key, method|
+        if method.respond_to?(:call)
+          hash[key] = instance_eval(&method)
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not -10) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)